<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitDialog</class>
 <widget class="QDialog" name="FitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fit geometry</string>
  </property>
  <property name="windowIcon">
   <iconset theme="view-fullscreen">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="actionLabel">
     <property name="text">
      <string>Action</string>
     </property>
     <property name="buddy">
      <cstring>actionBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="actionBox">
     <item>
      <property name="text">
       <string>Fit to geometry from file</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unify existing geometry</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="fileLabel">
     <property name="text">
      <string>File</string>
     </property>
     <property name="buddy">
      <cstring>toolButton</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="fileEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="xLabel">
     <property name="text">
      <string>X index</string>
     </property>
     <property name="buddy">
      <cstring>xSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="xSpin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="yLabel">
     <property name="text">
      <string>Y index</string>
     </property>
     <property name="buddy">
      <cstring>ySpin</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="ySpin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="radiusLabel">
     <property name="text">
      <string>Radius</string>
     </property>
     <property name="buddy">
      <cstring>radiusSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="radiusSpin">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>5.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>368</width>
       <height>107</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>actionBox</tabstop>
  <tabstop>fileEdit</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>xSpin</tabstop>
  <tabstop>ySpin</tabstop>
  <tabstop>radiusSpin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FitDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FitDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FitDialog</receiver>
   <slot>fitParametersChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FitDialog</receiver>
   <slot>fitParametersChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radiusSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FitDialog</receiver>
   <slot>fitParametersChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FitDialog</receiver>
   <slot>actionIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>FitDialog</receiver>
   <slot>openButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fitParametersChanged()</slot>
  <slot>actionIndexChanged(int)</slot>
  <slot>openButtonClicked()</slot>
 </slots>
</ui>
